v19.1/recommended-production-settings.md

```diff
@@ -240,6 +240,10 @@ $ cockroach start --cache=.25 --max-sql-memory=.25 <other start flags>
 Avoid setting `--cache` and `--max-sql-memory` to a combined value of more than 75% of a machine's total RAM. Doing so increases the risk of memory-related failures.
 {{site.data.alerts.end}}
 
+## Time zone data
+
+Certain features of CockroachDB require the [`tzdata`](https://www.iana.org/time-zones) package (sometimes called `tz` or `zoneinfo`), for example, to support using place names as time zone identifiers. Therefore, ensure that this package is installed and kept up-to-date on all VMs where CockroachDB nodes are running. 
```

Comment:
"machines", not VMs - we don't know or care whether they're virtualized. 

This does not apply to docker-based deployments. The tzdata package is included in the docker image and is update with each patch release of cockroachdb.

Reply:
Done. I'll work on one more commit adding the `tzdata` error to the common errors page.